DR: Purpose Statement

noelmace opened the pull request:
record the decision made in https://github.com/blindnet-io/communication-management/issues/86

milstan commented on docs/decision-records/DR-0003-purpose-statement.md:
```diff
@@ -0,0 +1,119 @@
+## Links
+
+- [Notions of Privacy](https://github.com/blindnet-io/product-management/blob/main/refs/notion-of-privacy/notion-of-privacy.md)
+
```
```suggestion
- [HBR: The Difference Between Purpose and Mission ](https://hbr.org/sponsored/2016/02/the-difference-between-purpose-and-mission)
```

milstan replied:
```suggestion
- [HBR: The Difference Between Purpose and Mission ](https://hbr.org/sponsored/2016/02/the-difference-between-purpose-and-mission)
```

milstan commented on docs/decision-records/DR-0003-purpose-statement.md:
```diff
@@ -0,0 +1,119 @@
+
+> « Helping developers make software that users love and trust. »
+
+A mission only factually state what an organization is **currently trying** to accomplish.
```
```suggestion
A mission only factually states **what** an organization is trying to accomplish.
```
(HBR says "Mission" is about WHAT, and "Purpose is about "WHY")

milstan replied:
```suggestion
A mission only factually states **what** an organization is trying to accomplish.
```
(HBR says "Mission" is about WHAT, and "Purpose is about "WHY")

milstan commented on docs/decision-records/DR-0003-purpose-statement.md:
```diff
@@ -0,0 +1,119 @@
+
+A mission only factually state what an organization is **currently trying** to accomplish.
+
+A purpose statement stays necessary to simply state why the organization exist as a collective, and what drives this collective on a daily basis.
```
```suggestion
A purpose statement stays necessary to simply state why the organization exist as a collective, and what drives this collective on a daily basis.
```
```suggestion
A purpose statement is necessary to  state **why** the organization exist as a collective, and what drives this collective on a daily basis.
```
HBR says "Mission" is about WHAT, and "Purpose is about "WHY"

milstan replied:
```suggestion
A purpose statement stays necessary to simply state why the organization exist as a collective, and what drives this collective on a daily basis.
```
```suggestion
A purpose statement is necessary to  state **why** the organization exist as a collective, and what drives this collective on a daily basis.
```
HBR says "Mission" is about WHAT, and "Purpose is about "WHY"

milstan commented on docs/decision-records/DR-0003-purpose-statement.md:
```diff
@@ -0,0 +1,119 @@
+- avoid any confusing, ambiguous or jargonistic term
+- be short (few words)
+- be easy to remember and say
+
```
```suggestion

The purpose statement is not:
-  slogan
- a vision statement
```

milstan replied:
```suggestion

The purpose statement is not:
-  slogan
- a vision statement
```

milstan reviewed (approved):
I made few suggestions for clarity around what is a purpose statement.

milstan commented:
@noelmace I have no idea why my suggestions appear twice. Feel free to just deal with one.

noelmace commented:
All suggestions from @milstan have been applied. I've just squashed the commits together and fixed a few typos.

This PR is ready to be merged directly by @milstan after @Vuk-BN's review.
